### DISCUSSION TITLE
Unions for the Costume Editor

### DISCUSSION THREAD

**Discussion by KojoBailey** [ScratchAddons/ScratchAddons#1402] [Ideas]
I'm not sure how possible this would be to add, but the ability to **merge, negate, and seperate** would be incredibly useful for the Scratch costume editor, and would allow for much easier shape manipulation.
![image](https://user-images.githubusercontent.com/50509420/105728517-35929080-5f67-11eb-89db-6be552185ee7.png)

**Comment by easrng** [comment]
If you want that, just use Inkscape.

**Reply by devxan** [reply]
Or Figma
